app/src/main/java/com/ihunda/android/binauralbeat/BBeat.java

```diff
@@ -1028,15 +1073,28 @@ private void stopProgram() {
         mToolbar.setTitle(getString(R.string.app_name));
 
         goToState(appState.SETUP);
+
```

Comment:
This code should be in handled in the goToState function as it's the master function to handle app state.

Reply:
okay.
Done